packages/apps/frontera/src/ui/overlay/Menu/Menu.tsx

```diff
@@ -18,7 +18,7 @@ export const MenuItem = forwardRef<HTMLDivElement, MenuItemProps>(
         {...props}
         ref={forwardedRef}
         className={twMerge(
-          'flex gap-2 items-center text-sm text-start py-[6px] px-[10px] leading-[18px] text-gray-700  rounded-sm outline-none cursor-pointer hover:bg-grayModern-100 hover:rounded-[4px] ',
+          'font-normal flex gap-2 items-center text-sm text-start py-[6px] px-[10px] leading-[18px] text-gray-700  rounded-sm outline-none cursor-pointer hover:bg-grayModern-100 hover:rounded-[4px] ',
```

Comment:
are you sure about this change? Yesterday i made with Steyn the design for the menu component

Reply:
yes, this is ensuring that line items won't ever inherit different font-weight from parent elements